source/about-compatibility.txt

```diff
@@ -0,0 +1,57 @@
+.. _about-driver-compatibility:
+
+============================
+MongoDB Compatibility Tables
+============================
+
+We include compatibility tables for each version of the driver to guide
+your decisions on what versions you need to ensure your environment
+remains fully operational.
+
+We maintain the following tables for each driver:
+
+- :ref:`MongoDB compatibility <mongodb-compatibility-tables>`
+- :ref:`Language compatibility <language-compatibility-tables>`
+
+Read the sections below for detailed explanations of each table.
+
+.. _mongodb-compatibility-tables
+
+MongoDB Compatibility Tables
+----------------------------
+
+In the **MongoDB Compatibility** tables, the columns are labeled with
+versions of MongoDB server and the rows are labeled with major release
+versions of the driver.
+
+The check marks (✓) indicate that the driver can access **all the
+features** of that specific version of MongoDB server unless those features 
+have been deprecated or removed. 
+
+If you are upgrading your server or driver version, you can use this table
+to identify the following:
+
+- version of the driver you need to access all the newest features
+  included in that version of the server
+
+- versions of the drivers tested against versions of the server
+
+.. note:: 
+   
+   Although the drivers do not advertise compatibility with the versions of
+   the server released after them, they are usually inter-operable.
+
+Avoid using older versions of the drivers that do not appear on the chart
+because they are not supported.
```

Comment:
nit: A single sentence that follows a note/figure at the end of a section tends to feel a bit lost or out of place IMO. Maybe we could move this into an `.. important::` or bump it up above the note somewhere?

Reply:
Ok. I haven't heard this feedback before, but can reorder sections. I also want to avoid two admonitions in a row if possible.